src/components/PollDetail/PollDetail.jsx

```diff
@@ -98,14 +99,15 @@ const Detail = () => {
   };
 
   const handleChange = () =>{
-    setDis(!dis)
+    setIsPulic(!isPublic)
```

Comment:
nên sử dụng callback nhé
```suggestion
    setIsPulic((prevState) => !prevState)
```